src/icareFhirMessaging.js

```diff
@@ -30,6 +30,17 @@ async function checkMessagingClient(messagingClient) {
   }
 }
 
+// This method is used to ensure AWS and FHIR messaging client are configured properly.
+// It is intended to be used with the --test-aws-auth flag. These functions are called separately during extraction.
+async function checkAwsAuthentication(config) {
+  // Ensure that we have the necessary aws info in the config file and that a MessagingClient can be made
+  const messagingClient = getMessagingClient(config);
+
+  // Ensure that the messagingClient can receive messages
+  await checkMessagingClient(messagingClient);
+  logger.debug('AWS authenticated properly');
```

Comment:
DM1 - Do you think this should be `logger.info` instead? Right now if you run with `--test-aws-auth` but without the debug flag, there's no indication that it succeeded.

Reply:
I went back and forth on this one, so I'll swap it back to `info` since you think it would be helpful too!

Reply:
Updated in 17a0e55!